npm/cypress-schematic/src/schematics/ng-generate/component/index.ts

```diff
@@ -0,0 +1,25 @@
+import { chain, externalSchematic, noop, Rule, SchematicContext, Tree } from '@angular-devkit/schematics'
+import cypressTest from '../cypress-test'
+import path = require('path');
+
+export default function (options: any): Rule {
+  return (_: Tree, _context: SchematicContext) => {
+    return chain([
+      externalSchematic('@schematics/angular', 'component', {
+        ...options,
+        skipTests: true,
+      }),
+      (tree: Tree, _context: SchematicContext) => {
+        const componentPath = tree.actions.filter((a) => a.path.includes(`${options.name}.component.ts`))
```

Comment:
This logic doesn't work when trying to generate a nested component e.g. `ng g c components/hello`. The component is generated, but a cypress test is not. This is because
```js
a = 'components/hello/hello.component.ts'
a.path.includes('components/hello.component.ts') === false
```

Is there a way to get the list of pending changes from the tree? Then we would just have to find a file ending in `.component.ts`

Reply:
Seems like this is failing on CI, taking a look...

Reply:
Is this supposed to generate a `cy.ts` file? It does not seem to work - tried locally. It does make a `spec` file, but I don't think that's the goal here, it is?

Reply:
It's passing on Angular-14 but not on Angular-13. I believe it is due to a problem with module resolution, similar to what we've seen with system-tests where it is resolving Angular deps from within the monorepo and not the scaffolded project.